check_osp_main.py

```diff
@@ -98,8 +98,15 @@ def main():
     args = parser.parse_args()
     # set logger
     logger = get_logger(args.local)
-    if float(args.warning) > float(args.critical):
```

Comment:
@kedark3 Since you are no longer changing the defaults, you can use this line in lieu of the conditions below.